VAMobile/src/constants/analytics.ts

```diff
@@ -1149,14 +1149,14 @@ export const Events = {
       name: 'vama_sm_start',
     }
   },
-  vama_snackbar_null: (location: string): Event => {
-    return {
-      name: 'vama_snackbar_null',
-      params: {
-        location,
-      },
-    }
-  },
+  // vama_snackbar_null: (location: string): Event => {
+  //   return {
+  //     name: 'vama_snackbar_null',
+  //     params: {
+  //       location,
+  //     },
+  //   }
+  // },
```

Comment:
Snooping out of curiosity from the design system side:
```suggestion
```
not sure there's value in retaining a commented out analytics event--one that in theory is hopefully eliminated with the new Snackbar since it was only added to track down a weird and rare crash where somehow the global Snackbar was not being initialized.

Reply:
I think I saw a few other events commented out and assumed this was done to keep the history in tact. If that's not necessary, then it can obviously be removed.

Reply:
This event has been removed.